src/main/kotlin/com/terraformation/backend/tracking/db/ObservationStore.kt

```diff
@@ -434,7 +434,7 @@ class ObservationStore(
               observationTypeId = newModel.observationType,
               plantingSiteId = newModel.plantingSiteId,
               startDate = newModel.startDate,
-              stateId = ObservationState.Upcoming,
+              stateId = newModel.state,
```

Comment:
Does it make sense to only allow `Upcoming` and `InProgress` or do we see a use case where it'd make sense to create an observation in one of the other states?